api/util/semver.py

```diff
@@ -0,0 +1,13 @@
+def is_semver(value: str) -> bool:
```

Comment:
@matthewelwell  may be worth removing this module and using the functions from https://github.com/Flagsmith/flagsmith-engine/pull/106/files

Reply:
Yes, that's a good idea for sure.